packages/cactus-plugin-ledger-connector-iroha2/src/main/typescript/web-services/query-endpoint.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * ExpressJS `query` endpoint
+ */
+
+import type { Express, Request, Response } from "express";
+
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+  IAsyncProvider,
+} from "@hyperledger/cactus-common";
+import {
+  IEndpointAuthzOptions,
+  IExpressRequestHandler,
+  IWebServiceEndpoint,
+} from "@hyperledger/cactus-core-api";
+import { registerWebServiceEndpoint } from "@hyperledger/cactus-core";
+
+import { PluginLedgerConnectorIroha2 } from "../plugin-ledger-connector-iroha2";
+import { safeStringifyException } from "../utils";
+
+import OAS from "../../json/openapi.json";
+
+export interface IQueryEndpointOptions {
+  logLevel?: LogLevelDesc;
+  connector: PluginLedgerConnectorIroha2;
+}
+
+export class QueryEndpoint implements IWebServiceEndpoint {
```

Comment:
@outSH Please make sure to include the Iroha version and the Cacti version in the name (it's not done consistently elsewhere in the code either so apologies for asking for this additional clarity).

I'd recommend `Iroha2QueryEndpointV1` but anything else will also do that makes it clear that this is a V1 endpoint of the framework that is used to communicate with V2 of Iroha ledgers.
I'll paste this same thing to a bunch of different places but the bottom line is the same for all of it: Over time it will get very confusing when you try to locate which file/class is where for which ledger/framework version. I know it might feel needlessly verbose now but the longer term it should pay dividends in reduced confusion and mistakes made when someone would try to use one in the place of the other.

Reply:
@petermetz Agree, that's why majority of names in this module have verbose names. I've simply overlooked this place - fixed :)

Reply:
@outSH Thank you very much! This helps a lot!